flask_resources/serializers/json.py

```diff
@@ -72,6 +73,55 @@ def serialize_object_list(self, obj_list):
         return json.dumps(obj_list, cls=self.encoder, **self.dumps_options)
 
 
+class CustomJSONSerializer:
+    """JSON serializer that allows extra information to be dumped."""
```

Comment:
It will require much more documentation about:
* why it is useful
* explanation of each param
* nice doc strings around